Acronym tag (#738)
Codecademy/docs: content/html/concepts/tags/terms/acronym/acronym.md

---
Title: '<acronym>'
Description: 'Creates text that will be displayed when hovered over.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Accessibility'
  - 'DOM'
  - 'Semantic'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

The `<acronym>` was used to highlight acronyms before it was deprecated. Now use the [`<abbr>`](https://www.codecademy.com/resources/docs/html/tags/abbr) tag instead. When using the tag there will be a dotted underline for the word to signify that it is an acronym.

>   

Reviewer: ```suggestion

```

> ## Syntax
> 
>   

Reviewer: ```suggestion
## Syntax

```

```html
<p>A great place to learn to code is <acronym title="Codecademy">CC</acronym><p>
```

>   

Reviewer: ```suggestion

```

The `title` attribute adds meaning of the acronym. It is not required but, when omitted, nothing will happen when it is hovered over.

>   

Reviewer: ```suggestion

```

## Example
  
```html
<p>
  Tomorrow I'll be heading to the Kennedy Space Center to learn about the
  National Aeronautics and Space Administration (NASA). Touring through the
  rockets used in
  <acronym title="National Aeronautics and Space Administration">NASA</acronym>
  will be such fun!
</p>
```

> ![Acronym example](https://raw.githubusercontent.com/Codecademy/docs/main/media/abbr-example.png)

Reviewer: ```suggestion

![Acronym example](https://raw.githubusercontent.com/Codecademy/docs/main/media/abbr-example.png)
```

---
Title: '<acronym>'

> Description: 'Creates text when hovered over the text will be displated'

Reviewer 2: ```suggestion
Description: 'Creates text that will be displayed when hovered over.'
```

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Accessibility'
  - 'DOM'
  - 'Semantic'
  - 'Tags'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---
  
## `<acronym>` tag

> The `<acronym>` was used the make acronyms before it was deprecated. Now use the [`<abbr>`](https://www.codecademy.com/resources/docs/html/tags/abbr) instead. When using the tag there will be a dotted underline for the word to signify that it is an acronym.

Reviewer 2: ```suggestion
The `<acronym>` was used to highlight acronyms before it was deprecated. Now use the [`<abbr>`](https://www.codecademy.com/resources/docs/html/tags/abbr) instead. When using the tag there will be a dotted underline for the word to signify that it is an acronym.
```

  
## Syntax

> Use the `title` attribue to add the meaning of the acronym but it is not required but nothing will happen when it is hovered over.

Reviewer 2: ```suggestion
Use the `title` attribute to add the meaning of the acronym. It is not required but, when omitted, nothing will happen when it is hovered over.
```

> Use the `title` attribute to add the meaning of the acronym. It is not required but, when omitted, nothing will happen when it is hovered over.

Reviewer: ```suggestion
```

Let's move this below the snippet.

  
```html
<p>A great place to learn to code is <acronym title="Codecademy">CC</acronym><p>
```
  

> ## Example

Reviewer 2: This is an `<abbr>` example. It should be an `<acronym>` example, with a comment noting it's depreciated.

>   

Reviewer: ```suggestion
  
The `title` attribute adds meaning of the acronym. It is not required but, when omitted, nothing will happen when it is hovered over.
  
```

  
```html
<p>
  Tomorrow I'll be heading to the Kennedy Space Center to learn about the
  National Aeronautics and Space Administration (NASA). Touring through the
  rockets used in
  <acronym title="National Aeronautics and Space Administration">NASA</acronym>
  will be such fun!
</p>
```

>   

Reviewer: ```suggestion

```

One more, here. Hopefully, this will allow the "format" check to pass.

> ![Abbr example](https://raw.githubusercontent.com/Codecademy/docs/main/media/abbr-example.png)

Reviewer 2: This is an `<abbr>` example. It should be an `<acronym>` example.
